### ISSUE TITLE
When using DecoratedFileUploadImplClick, the native upload widget is visible

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#120]
```
In an IE build, the native upload widget displays 300 pixels above and 300 
pixels to the left of the decorated upload widget. A simple fix is to use a 
larger offset (e.g. 10000) instead of 300.
```

Original issue reported on code.google.com by `ch.mar...@gmail.com` on 14 Sep 2011 at 11:05

**Comment by GoogleCodeExporter** [comment]
```
which version are you using?
```

Original comment by `manuel.carrasco.m` on 22 Sep 2011 at 7:31

**Comment by GoogleCodeExporter** [comment]
```
0.6.3
```

Original comment by `ch.mar...@gmail.com` on 22 Sep 2011 at 7:49

**Comment by GoogleCodeExporter** [comment]
```
fixed in r=933, please use 0.6.4-SNAPSHOT
```

Original comment by `man...@talkwheel.com` on 14 Oct 2011 at 8:32

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 14 Oct 2011 at 8:50
- Changed state: **Fixed**
